### DISCUSSION TITLE
testing slack integration

### DISCUSSION THREAD

**Discussion by drin** [skyhookdm/.github#10] [General]
creating this discussion to check if the slack integration is subscribed correctly

**Comment by drin** [comment]
well, it looks like `/github subscribe skyhookdm discussions` does not actually pick up new discussions in the organization

**Comment by drin** [comment]
closing as ineffective for the desired result

**Reply by carlosmalt** [reply]
Ha -- didn't see this until now. Bummer. I guess I was able to reproduce your result 😄
